code/game/objects/items/spear.dm

```diff
@@ -130,7 +130,15 @@
 	. = ..()
 	if(!proximity)
 		return
-	if(wielded)
+	if(wielded && istype(AM))
```

Comment:
you could early return this

Reply:
I don't really see what the point of that would be.

Reply:
to not have 12 lines of code one tab further than it could be, also this https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-early-returns

Reply:
oh, you're saying to early return the wielded part as well? okay.